bool left_front
bool right_front
bool left_middle
bool right_middle
bool left_rear
bool right_rear
